extensions/boot/src/main/java/org/vaadin/spring/boot/AtmosphereAutoConfiguration.java

```diff
@@ -20,7 +20,7 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.InitializingBean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
-import org.springframework.boot.context.embedded.ServletListenerRegistrationBean;
+import org.springframework.boot.web.servlet.ServletListenerRegistrationBean;
```

Comment:
This change might actually be good even for the 0.0.X series, for those who wish to use spring boot 1.5 with Vaadin 7. This was in a separate commit, so it can be cherry picked to V7 branch later too if needed.